README.md

```diff
@@ -1,253 +1,239 @@
-* If "percentageOfResourcesReturned" configuration is not set to default (100), resources returned are rounded down when percentage is calculated. Valid settings for this config is between 0-100.
-
-## Building
-* Remove Building "Invalid Placement" restriction
-* Remove Building Object deterioration by weather.
-* Remove Building Object deterioration by water.
-* Free rotation mode for the default Building Mode.
-* Advanced Building Mode
-* Advanced Editing Mode
-* Structural Integrity modification system
-* Modify the maximum distance you can place objects at
-* Allow the repair function of the hammer tool to repair all placeables in an area.
-* Allow placeables to drop full material cost upon being destroyed/dismantled, even if built by someone else in multiplayer.
+### Inventory
+* Modify the inventory behavior so that items will be placed in the first slot rather than the last available slot.
+* Reconfigure the amount of rows and columns in player inventory and containers.
+* Modify the size of player inventory and all types of containers, including vehicles:
+Player inventory can be extended up to 20 x 8.
+Wood chest inventory can be extended up to 10 x 8.
+Iron chest inventory can be extended up to 20 x 8 (this setting also applies to the inventory of Longships and Carts and a scrollbar will appear automatically when the container size exceeds a certain limit).
+**Note: As of 0.9.4, player inventory slot configuration is not compatible with Equipment and Quick slots mod**
+
+
+### Items
+* Modify the durability of each item type separately.
+* Modify the amount of armor granted by armor pieces.
+* Modify the amount of damage blocked by all shields.
+* Remove teleport prevention from all items.
+* Reduce the weight of all items by percent.
+* Modify maximum item stack size by percent.
+
+
+
+# Crafting and Production
+
+### Workbench
+* Modify Workbench radius.
+* Modify the radius at which Workbench attachments can be placed.
+* Allow crafting stations to automatically repair all appropriate items in the player's inventory on interaction.
+
+### Kiln and Furnace
+* Modify Kiln and Furnace processing speed.
+* Modify kiln and Furnace maximum capacity.
+* Allow items produced by Kiln and Furnace to be automatically placed in the closest container.
+
+### Beehive 
+* Modify Beehive honey production speed.
+* Modify Beehive capacity.
+* Display time left until next production.
+* Allow items produced by Beehive to be automatically placed in the closest container.
+
+### Fermenter
+* Modify Fermenter speed.
+* Modify Fermenter output amount.
+* Display time left until next production.
+
+## Deconstruct feature
+* This feature allows dismantling items craftable in that station back into their original components. It is accessed via an additional tab in every different crafting station (except cauldron). There are some added functionality and restrictions:
```

Comment:
Could we break this paragraph out into different lines?

Reply:
I've done all three requested changes, added the more recent changes to the readme (including player visibility, njord and mouse position) and went through additional formatting and read proofing to make sure it all looks as neat as possible. Let me know if it needs any further editing!